ci-operator/step-registry/redhat-appstudio/health-check/redhat-appstudio-health-check-commands.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+export GITHUB_TOKEN
+GITHUB_TOKEN=$(cat /usr/local/ci-secrets/redhat-appstudio-qe/github-token)
+
+cd "$(mktemp -d)"
+git clone --branch main https://github.com/redhat-appstudio/qe-tools .
+make build
+./qe-tools prowjob health-check --notify-on-pr=true --fail-if-unhealthy=false
```

Comment:
for now the pre-step will only inform the PR author about the outage, but **won't fail the job completely**

that behaviour is planned to be enabled after we evaluate this tool to be working reliably